### PR TITLE
Ref#44672 make test product controller

### PR DESCRIPTION
**Pull request by hoangtrungthong**
## Related Tickets:
- [#44672](https://edu-redmine.sun-asterisk.vn/issues/44672)

## WHAT this PR does?
make  test product controller
## Notes Impacted Areas:
- N/A.

## Attachments:
image

### CODE REVIEW

**Comment by hoangtrungthong** [comment]
ready

**Comment by hoangtrungthong** [comment]
ready

File: app/Http/Controllers/Admin/ProductController.php [hunk 1]
```diff
@@ -114,16 +114,14 @@ public function store(StoreProductRequest $request)
...
-                        ]
-                    );
-                }
+            foreach ($request->images as $file) {
+                $img = uploadFile('images', config('path.PRODUCT_UPLOAD_PATH'), $request, $file);
+                $this->productImageRepository->create(
+                    [
+                    'product_id' => $product->id,
+                    'path' => $img,
+                    ]
```
Comment (ghost) [inline app/Http/Controllers/Admin/ProductController.php]:
```suggestion
                    [
                        'product_id' => $product->id,
                        'path' => $img,
                    ]
```
fix tương tự các chỗ khác nha

**Comment by hoangtrungthong** [comment]
ready

**Comment by hoangtrungthong** [comment]
ready

**Pull request merged**